src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -0,0 +1,672 @@
+Even though the duplication requires additional effort to ensure the consistency of the data, it provides several advantages:
+- It makes the dataset easier for humans to scan, as essential information is easily accessible without loading the data files.
+- The dataset adheres to the BIDS standard and can benefit from tools built on top of this standard, such as [bids-validator](https://github.com/bids-standard/bids-validator).
+- It simplifies the separation of data and basic metadata, enabling, for example, the publication of a dataset in a lightweight fashion with access to the data files on request (as implemented by [DataLad](https://www.datalad.org)).
+
+
+##
+
+<!--
+This block generates a filename templates.
+The inputs for this macro can be found in the directory
+  src/schema/rules/files/raw
+and a guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+
+<!-- Link Definitions -->
+
+{{ MACROS___make_filename_template(
+   "raw",
+   datatypes=["icephys"],
+   suffixes=["icephys", "events", "channels", "electrodes","scans","probes","coordsystem"]
+)
+}}
+
+{{ MACROS___make_filename_template(
+   "raw",
+   datatypes=["ecephys"],
+   suffixes=["ecephys", "events", "channels", "electrodes","scans","probes","coordsystem"]
+)
+}}
+
+
+#  PARTICIPANT keyfiles
+## Participant information
+
+The `participants.tsv` file is located at the root of the data set directory.
+Its presence is RECOMMENDED in order to describe information about the individual subjects (animals) from which the data was recorded.
+It follows the [general BIDS specifications to describe participants](../modality-agnostic-files.md#participants-file).
+
+On top of the existing columns that can be present in this file and that are described in the BIDS specifications (`participant_id`, `species`, `strain`, `strain_rrid`, `sex`, `handedness`, and `age`), we propose to allow adding the following ones:
+
+
+#  EPHYS specific files
+
+The following metadata files are REQUIRED for a given animal ephys session:
+
+1. `[_ses-<session_label>]_probes.tsv`: A REQUIRED file listing information on the device used to acquire the electrophysiology data, such as implant or probe specification, location, material, and others.
+2. `[_ses-<session_label>]_electrodes.tsv`: A REQUIRED file listing information on the points of electrical contact to the tissue, such as impedance, names, relative positions, and others.
+3. `[_ses-<session_label>]_channels.tsv`: A REQUIRED file listing information on the recorded signals, such as preprocessing, filtering, ids, and others.
+
+As with all tsv-based metadata files in BIDS the probes, electrodes and channels tsv files can be accompanied by json sidecar files.
+
+
+## Coordinate System JSON (`*_coordsystem.json`) & Photos of electrode positions (`_photo.jpg`)
+
+This file provides metadata on the global coordinate system in which the electrodes are placed.
```

Comment:
```suggestion
This file provides metadata on the coordinate system in which the electrodes are placed.
```

I think the word "global" is confusing here. Is it serving a purpose I am missing?